site/content@beta/backend-apis/about-apis.md

```diff
@@ -0,0 +1,435 @@
+---
+weight: 20
+---
+
+# About our APIs
+
+There are two primary Chat APIs:
+
+- [Agent Chat API](https://developers.livechatinc.com/beta-docs/agent-chat-api/), which serves to join a chat as an agent.
+- [Customer Chat API](https://developers.livechatinc.com/beta-docs/customer-chat-api/), which serves to join a chat as a customer.
+
+The **Agent Chat API** contains a different set of methods than the **Customer Chat API**. We separated both APIs as they cover different use cases. 
+
+To use those APIs, you need to be _authorized_. This topic is thoroughly explained in the [**Authorizing API calls**](../authorization) document.
+
+## RTM API vs. Web API
+
+Both APIs have much in common. One similarity is that they're both **Real Time Messaging APIs (RTM APIs)**, which translates to enabling real-time communication. Websocket transport allows for **pushes**, which are server-client methods used to keep application state up-to-date. In case of the Agent App and the Chat Widget, the continuous connection is crucial. That's why they both implement RTM API.
+
+|**Characteristics**|     **RTM API**      | **Web API**               |
+|:-----------------:|:--------------------:|:-------------------------:|
+|**connection type**|   websocket          |  independent requests     |
```

Comment:
websocket to 1 z kilku możliwości, lepiej ogólnie - stałe połączenie, np. stateful connection (e.g. websocket)